docs/execution-environments/r-support.rst

```diff
@@ -1,178 +1,278 @@
+- see hardware consumption and stdout/stderr output during training
+- log data versions and other properties
+- log model weights and other artifacts
+- log performance charts
+- tage experiments to organize them
 
-**Python environment setup**
+.. note::
 
-Under the hood, Neptune uses reticulate, which in turn calls Python.
-That means you must:
+    This integration is tested with ``R version 3.6.3``, ``neptune_0.1.0``, and ``Python 3.6.12``.
 
-* Install Python in your environment (venv, conda, miniconda and pure Python environments are supported).
-* Point to that environment when you initialize Neptune, as shown below.
+    Remember that even if you are using R you still need to have Python installed on your system.
 
-Initialize Neptune
-------------------
-Toward the top of your script, you must import the ``neptune`` library and initialize the connection with Neptune.
+Where to start?
+---------------
+To get started with using Neptune client for R, follow the :ref:`quickstart <r-quickstart>` below.
 
-Specify:
+You can also skip the basics and take a look at how to log model weights and performance charts in the :ref:`more options <r-more-options>` section.
 
-* ``project_name``: the name of your project in Neptune
-* ``api_token``: a key associated with your Neptune account
+If you want to try things out right away you can either:
 
-.. code:: R
+|colab-script-neptune|
 
-    library('neptune')
-    init_neptune(project_name = 'shared/r-integration',
-                 api_token = 'ANONYMOUS')
+.. _r-quickstart:
 
-Once this is added, you can start logging your experiment data to Neptune.
+Quickstart
+----------
+This quickstart will show you how to:
 
-.. note:: As an example, we are using the 'ANONYMOUS' user token and a public project named 'shared/r-integration'.
+* Install the necessary Neptune package from CRAN
+* Connect Neptune to your training code and create the first experiment
+* Log metrics and hardware consumption to Neptune
+* Explore learning curves in the Neptune UI
 
-.. warning:: The suggested way to pass in your ``api_token`` is to store your key in an environment variable and pass it using ``Sys.getenv('MY_NEPTUNE_KEY')``.
+.. _r-before-you-start:
 
-    .. code:: R
+Before you start
+^^^^^^^^^^^^^^^^
+To log experiments to Neptune from R you need to satisfy the following prerequisites:
 
-        Sys.setenv('NEPTUNE_API_TOKEN'='eyJhcGlfYWRkcmVzcyI6Imh0dHBz')
+* Have ``Python 3.x`` installed. The easiest way to |install Python environment is with Conda|.
 
-    .. code:: R
+.. tip::
 
-        init_neptune(project_name = 'my_workspace/my_project',
-                     api_token = Sys.getenv('NEPTUNE_API_TOKEN')
-                     )
+    If you are using R studio you can just follow along and you will be prompted by the console to install miniconda.
 
-**Specifying Python environment**
+Step 1: Install Neptune package
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Install ``neptune`` package for R from CRAN.
```

Comment:
Make sure to note that this should be done within R interactive console, not bash - we had a user that missed that

Reply:
ok, cool thanks